src/ORM/ListDecorator.php

```diff
@@ -133,12 +133,19 @@ public function last()
         return $this->list->last();
     }
 
+    public function getTotalItems()
```

Comment:
Can't use `int` as the return type because that would require changing it in `PaginatedList::getTotalItems()` as well, which is a breaking change